Essentials/src/messages.properties

```
@@ -692,6 +692,13 @@ setworthCommandUsage=/<command> [itemname|id] <price>
 sheepMalformedColor=\u00a74Malformed color.
 shoutFormat=\u00a76[Shout]\u00a7r {0}
 showkitCommandDescription=Show contents of a kit.
+editsignCommandDescription=Edits a sign in the world.
+editsignCommandLimit=\u00a74Your provided text is too big to fit on the target sign.
+editsignCommandNoLine=\u00a74You must enter a line number between \u00a7c1-4\u00a74.
+editsignCommandResetLine=\u00a76Reset line\u00a7c {0}\u00a76 on your target sign.
```

reset -> clear for clarity since command changed, also probably want a message for whole sign
```suggestion
editsignCommandClear=\u00a76Cleared your target sign.
editsignCommandClearLine=\u00a76Cleared line\u00a7c {0}\u00a76 on your target sign.
```